src/docker/10-debianconsole/.prebuild.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -e
 
+# FIXME Remove this when Docker supplies multi-arch base images
+if [ "$ARCH" == "arm" ]; then
+  BASE_IMAGE=debian:jessie
+  ARM_BASE_IMAGE=armhf/debian:jessie
```

Comment:
Ah, finally I understand. `resin/rpi-raspbian` builds on Raspbian, which is specially compiled [to support ARMv6 including hard-float](https://www.raspbian.org/RaspbianFAQ#What_is_Raspbian.3F) (didn't know that). For the RPi1, that would be the best solution. And it works for newer devices as well.

So we could use `resin/rpi-raspbian` for the debian console and `armhf/ubuntu` for the ubuntu one (with the latter one not working on RPi1).